docs/source/usersguide/plots.rst

```diff
@@ -0,0 +1,137 @@
+.. _usersguide_plots:
+
+======================
+Geometry Visualization
+======================
+
+.. currentmodule:: openmc
+
+OpenMC is capable of producing two-dimensional slice plots of a geometry as well
+as three-dimensional voxel plots using the geometry plotting :ref:`run mode
+<usersguide_run_modes>` is a geometry plotting mode. The geometry plotting mode
+relies on the presence of a :ref:`plots.xml <io_plots>` file that indicates what
+plots should be created. To create this file, one needs to create one or more
+:class:`openmc.Plot` instances, add them to a :class:`openmc.Plots` collection,
+and then use the :class:`Plots.export_to_xml` method to write the ``plots.xml``
+file.
+
+-----------
+Slice Plots
+-----------
+
+.. image:: ../_images/atr.png
+   :width: 300px
+
+By default, when an instance of :class:`openmc.Plot` is created, it indicates
+that a 2D slice plot should be made. You can specify the origin of the plot
+(:attr:`Plot.origin`), the width of the plot in each direction
+(:attr:`Plot.width`), the number of pixels to use in each direction
+(:attr:`Plot.pixels`), and the basis directions for the plot. For example, to
+create a x-z plot centered at (5.0, 2.0, 3.0) with a width of (50., 50.) and
+400x400 pixels::
+
+  plot = openmc.Plot()
+  plot.basis = 'yz'
+  plot.origin = (5.0, 2.0, 3.0)
+  plot.width = (50., 50.)
+  plot.pixels = (400, 400)
+
+The color of each pixel is determined by placing a particle at the center of
+that pixel and using OpenMC's internal ``find_cell`` routine (the same one used
```

Comment:
I think it would be worthwhile to fully document the PyAPI's `find_cell` routine here, or in the geometry page, including with an example. That method can be insanely useful